_resource/overrides/main.html

```diff
@@ -14,7 +14,8 @@ <h3>Contact us </h3>
        <p>For free technical help, visit the Percona <a class="reference external" href="https://forums.percona.com/c/mysql-mariadb/percona-xtrabackup/10?utm_campaign=Doc%20pages" target="_blank">Community Forum</a>.<br>
         <p>To report bugs or submit feature requests, open a <a class="reference external" href="https://jira.percona.com/projects/PXB/issues/" target="_blank">JIRA</a> ticket.<br>
          <p>For paid <a class="reference external" href="https://www.percona.com/services/support"> support </a> and <a class="reference external" href="https://www.percona.com/services/managed-services"> managed </a> or <a class="reference external" href="https://www.percona.com/services/consulting">consulting services </a>, contact <a class="reference external" href="https://www.percona.com/about-percona/contact" target="_blank">Percona Sales</a>.</p>
- </article>
+         <img referrerpolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=d1635340-c352-40f2-8061-0d38bcb6eccf" />
+        </article>
```

Comment:
This is how we add the pixel to each page. This link is unique per product. Check the spreadsheet.